packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -98,8 +99,22 @@ class Route extends EmberObject implements IRoute {
   controller!: Controller;
   currentModel: unknown;
 
+  _bucketCache: BucketCache | undefined;
   _internalName!: string;
   _names: unknown;
+  _router: EmberRouter | undefined;
+
+  constructor() {
+    super(...arguments);
+
+    let owner = getOwner(this);
```

Comment:
I think `owner` is passed as the first argument to `constructor`, so you can use that instead of calling `getOwner(this)`.